src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -938,37 +938,40 @@ users:
 ----
 It is advised to embed credentials and certificates in the configuation file, instead of pointing to files on the local file system.
 
-- *nodeJarDownloadCommand:* command which will be executed inside the pod to download *ProActive libraries*. Thanks to this command, ProActive libraries are downloaded when the pod starts and prior installation of the libraries inside the Docker image is not required.
-This command needs to download using the http(s) protocol the `node.jar`from the ProActive Resource Manager server.
-+
-Examples:
-....
-wget -P /opt/proactive/ http://192.168.99.1:8080/rest/node.jar
+- *kubernetesNamespace:* (Optional) Kubernetes namespace if different from _default_ namespace.
 
-wget -P /opt/proactive/ --no-check-certificate https://192.168.99.1:8443/rest/node.jar
-....
+- *cleaningDelaySeconds:* (Optional) Cleaning delay, in seconds until cleaning service starts. Default value is 120 seconds.
+
+- *cleaningFrequencySeconds:* (Optional) Cleaning frequency, Kubernetes node cleaning is done every x seconds. Default value is 30 seconds.
 
-- *rmCredentials:* path to a file which contains the credentials of an administrator user which will own the node source.
 - *scale:* Number of pods (ProActive Nodes) to create. This parameter is used when the <<../admin/ProActiveAdminGuide.html#_node_source_policies,Node Source Policy>> is not a <<../admin/ProActiveAdminGuide.html#_dynamic_policy,Dynamic Policy>>.
-- *javaNodeJarCommand:* command used to start the ProActive Node. The default value needs to be modified according to the location of the java command.
+
+- *image:* Docker image used to deploy ProActive Nodes. Example: java:8-jre
```

Comment:
if image empty. Does it take java:8-jre as default image?

Reply:
Yes, it is the case. I'll add it.

Reply:
@tao-xinxiu add (Optional) as well